controls/SV-235158.rb

```diff
@@ -140,12 +140,11 @@
       it { should_not be_empty }
     end
 
-    sql_session.query(query_firewall_users).results.rows.each do |fw_user|
-      describe "USERHOST #{fw_user['userhost']}" do
-        subject { fw_user }
-        its(['mode']) { should match /LEARNING|DETECTING|PROTECTING/ }
-      end
```

Comment:
@ejaronne The current targets that we have (both hosted and AWS RDS) don't have the `MYSQL_FIREWALL_USERS` table. Should we consider enabling the MySQL Enterprise Firewall to test this control?